{{cookiecutter.project_slug}}/scripts/init-github.sh

```diff
@@ -6,6 +6,7 @@
 git init
 git add .
 git commit -m "Initial commit"
+# git remote add origin git@github.com:thinknimble/{{cookiecutter.project_slug}}.git
 gh repo create thinknimble/{{cookiecutter.project_slug}} --private -y
 git push origin main
 printf "\033[0;32mRepo https://github.com/thinknimble/{{cookiecutter.project_slug}}/\033[0m \n"
```

Comment:
Should improve this later, but for those who are creating the repo first on Github (and who don't want to install the `gh` utility), the comment here is the alternative approach.

Reply:
side comment: All these scripts assume you are pushing repos to `thinknimble`. That should not be an assumption.